test/integration/pubsub/test_pubsub.rb

```diff
@@ -0,0 +1,153 @@
+require "helpers/integration_test_helper"
+require "securerandom"
+require "base64"
+
+class TestPubsubService < FogIntegrationTest
+  @@client = Fog::Google::Pubsub.new
```

Comment:
You don't really gain much here from a class variable, and they are considered bad style in ruby, so it'd be better to just create an instance variable.